prow/scripts/cluster-integration/kyma-gke-minio-gateway-migration.sh

```diff
@@ -0,0 +1,399 @@
+    shout "Create DNS Record for Apiserver proxy IP"
+    date
+    APISERVER_IP_ADDRESS=$(kubectl get  service -n kyma-system apiserver-proxy-ssl -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+    APISERVER_DNS_FULL_NAME="apiserver.${DNS_SUBDOMAIN}.${DNS_DOMAIN}"
+    CLEANUP_APISERVER_DNS_RECORD="true"
+    IP_ADDRESS=${APISERVER_IP_ADDRESS} DNS_FULL_NAME=${APISERVER_DNS_FULL_NAME} "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/create-dns-record.sh"
+fi
+
+"${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/get-helm-certs.sh"
+
+MINIO_HOST=$(kubectl -n kyma-system get endpoints assetstore-minio -o jsonpath='{.subsets[0].addresses[0].ip}' | xargs -n1 echo)
+MINIO_PORT=$(kubectl -n kyma-system get service assetstore-minio -o jsonpath='{.spec.ports[0].port}' | xargs -n1 echo)
+ACCESS_KEY=$(kubectl get secret assetstore-minio -n kyma-system -o jsonpath="{.data.accesskey}" | base64 -D | xargs -n1 echo)
+SECRET_KEY=$(kubectl get secret assetstore-minio -n kyma-system -o jsonpath="{.data.secretkey}" | base64 -D | xargs -n1 echo)
+CONTENT_TYPE="application/octet-stream"
+
+# Creates sample file and uploads it to minio.
+# arg1: the name of the bucket the file will be uploaded to
+# arg2: the name of the created file
+function upload_sample_file_to_minio {
+    set -u
+    BUCKET_NAME=$1
+    FILE_NAME=$2
+    RESOURCE="${BUCKET_NAME}"/"${FILE_NAME}"
+    DATE=$(date -R)
+    SIGNATURE=PUT"\n\n${CONTENT_TYPE}\n${DATE}\n/${RESOURCE}"
+    CHECKSUM=$(echo -en "${SIGNATURE}" | openssl sha1 -hmac "${SECRET_KEY}" -binary | base64)
+    echo "sample" | curl -X PUT -d @- \
+        -H "Date: ${DATE}" \
+        -H "Content-Type: ${CONTENT_TYPE}" \
+        -H "Authorization: AWS ${ACCESS_KEY}:${CHECKSUM}" \
+        http://"${MINIO_HOST}":"${MINIO_PORT}"/"${RESOURCE}"
+    set +u
+}
+
+# upload samples to minIO
+PUBLIC_BUCKET=$(kubectl -n kyma-system get configmap asset-upload-service -o jsonpath="{.data.public}" | xargs -n1 echo)
+PRIVATE_BUCKET=$(kubectl -n kyma-system get configmap asset-upload-service -o jsonpath="{.data.private}" | xargs -n1 echo)
+
+upload_sample_file_to_minio "${PUBLIC_BUCKET}" sample/sample
+upload_sample_file_to_minio "${PUBLIC_BUCKET}" sample
+
+upload_sample_file_to_minio "${PRIVATE_BUCKET}" sample/sample
+upload_sample_file_to_minio "${PRIVATE_BUCKET}" sample
+
+# switch to minIO GCS gateway mode
+ASSET_STORE_RESOURCE_NAME="asset-store-overrides"
+
+kubectl create -n kyma-installer secret generic "${ASSET_STORE_RESOURCE_NAME}" --from-file=minio.gcsgateway.gcsKeyJson="${GOOGLE_APPLICATION_CREDENTIALS}"
+kubectl label -n kyma-installer secret "${ASSET_STORE_RESOURCE_NAME}" "installer=overrides" "component=assetstore" "kyma-project.io/installation="
+
+"${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/create-config-map.sh" --name "${ASSET_STORE_RESOURCE_NAME}" \
+    --data "minio.persistence.enabled=false" \
+    --data "minio.gcsgateway.enabled=true" \
+    --data "minio.defaultBucket.enabled=false" \
```

Comment:
It is no longer needed as it is disabled by default. Can you ale update `kyma-gke-minio-gateway` job and tutorial?